feature-libs/product-configurator/rulebased/components/add-to-cart-button/configurator-add-to-cart-button.component.ts

```diff
@@ -348,6 +348,18 @@ export class ConfiguratorAddToCartButtonComponent implements OnInit, OnDestroy {
         CommonConfigurator.OwnerType.SAVED_CART_ENTRY
       ) {
         this.goToSavedCartDetails(container.routerData.owner);
+      } else if (
+        container.routerData.owner.type ===
+        CommonConfigurator.OwnerType.CART_ENTRY
+      ) {
+        this.routingService.go({ cxRoute: 'cart' });
```

Comment:
that's wrong, we might have initiated the navigation from checkout. Please check the behaviour in develop

Reply:
@ChristophHi,
if this condition is removed, then the navigation for `cartEntry` does not work anymore.
Instead of navigating to the cart the end-user is navigated to the checkout review process.

Reply:
It's not about removing this line :-) but the process should work as before